### PR TITLE
Initial ECS cluster support

### PR DESCRIPTION
**Pull request by thoo5ieb**

### CODE REVIEW

**Comment by thoo5ieb** [comment]
I was thinking about the encapsulation of `ecs.Cluster` to make the interface simpler. But before, I would like to hear what you think once you get time to take a look at this.

File: modules/aws/ecs.go [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		},
+	}
+	output, err := client.DescribeClusters(input)
+	if err != nil {
+		return nil, err
+	}
+
+	numClusters := len(output.Clusters)
+	if numClusters != 1 {
+		return nil, fmt.Errorf("Couldn't find ECS cluster %v in region %v", name, region)
```
Comment (brikis98) [inline modules/aws/ecs.go]: Might be clearer to say: "Expected to find 1 ECS cluster named '%s" in region '%v', but found '%d'

File: modules/aws/ecs_test.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateEcsClusterE(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+	ecs, err := CreateEcsClusterE(t, region, "terratest")
+	defer DeleteEcsCluster(t, region, ecs)
```
Comment (brikis98) [inline modules/aws/ecs_test.go]: Call `GetEcsCluster` in between?

Comment (thoo5ieb) [reply modules/aws/ecs_test.go]: Done.

File: modules/aws/ecs.go (hunk 1 above)
Comment (thoo5ieb) [reply modules/aws/ecs.go]: Updated, thanks!

File: modules/aws/ecs_test.go (hunk 2 above)
Comment (thoo5ieb) [reply modules/aws/ecs_test.go]: Thanks!

**Comment by thoo5ieb** [comment]
Great, happy to hear it! :-)

**Comment by thoo5ieb** [comment]
Tested it locally on my account, I had no problem. In any case, you will encounter anything, just let me know!

**Pull request merged**